client/components/global-notices-v2/style.scss

```diff
@@ -0,0 +1,50 @@
+.global-notices-v2 {
+	text-align: right;
+	pointer-events: none;
+	z-index: z-index("root", ".global-notices");
+	position: fixed;
+	bottom: var(--masterbar-height);
+	right: 0;
+	left: 0;
+
+
+	@include breakpoint-deprecated( ">660px" ) {
+		bottom: calc(var(--masterbar-height) + 16px);
+		right: 16px;
+		left: auto;
+		max-width: calc(100% - 32px);
+	}
+
+	@include breakpoint-deprecated( ">960px" ) {
+		bottom: calc(var(--masterbar-height) + 24px);
+		right: 24px;
+		max-width: calc(100% - 48px);
+	}
+
+	@include breakpoint-deprecated( ">1040px" ) {
+		right: 32px;
+		max-width: calc(100% - 64px);
+	}
+}
+
+.global-notices-v2 .components-snackbar {
```

Comment:
Let's avoid using `.components-*` class names, as they are meant to be internal implementation details.